code/controllers/subsystem/vote.dm

```diff
@@ -387,14 +391,16 @@ SUBSYSTEM_DEF(vote)
 				var/votedesc = capitalize(mode)
 				if(mode == "custom")
 					votedesc += " ([question])"
-				log_and_message_admins("cancelled the running [votedesc] vote.")
+				log_and_message_admins("cancelled the running '[votedesc]' vote.")
 				reset()
 		if("toggle_restart")
 			if(admin)
 				GLOB.configuration.vote.allow_restart_votes = !GLOB.configuration.vote.allow_restart_votes
+				log_and_message_admins("[key_name_hidden(usr)] has [GLOB.configuration.vote.allow_restart_votes ? "enabled" : "disabled"] public restart voting.")
```

Comment:
Remove the double key name here
```dm
/proc/log_and_message_admins(message)
	log_admin("[key_name(usr)] " + message)
	message_admins("[key_name_admin(usr)] " + message)
```

Reply:
You know, I actually checked that proc twice to make sure it was actually doing what I wanted. Not sure how I missed that.